app/components/show/controls_component.html.erb

```diff
@@ -38,8 +38,8 @@
       </button>
       <ul class="dropdown-menu" aria-labelledby="dropdown-descriptive">
         <% if catalog_record? %>
-          <li><%= link_to 'Refresh', refresh_metadata_item_path(id: druid),
-                                     class: "dropdown-item #{'disabled' if button_disabled?}",
+          <li><%= link_to refresh_button_label, refresh_metadata_item_path(id: druid),
+                                     class: "dropdown-item #{'disabled' if button_disabled? || Settings.ils_cutover_in_progress}",
```

Comment:
Why not put this in `button_disabled?` method instead of the ERB?

Reply:
I would have preferred that, but it looks like `button_disabled?` governs the behavior of a few buttons, and we only want to disable one of them.